oasis/app.py

```diff
@@ -71,7 +77,7 @@ def stop_loop(deadline):
     def shutdown():
         logger.info('Stopping http server')
         server.stop()
-        JobHandler.close_all_jobs()
+        handler.close()
```

Comment:
close -> stop?

Reply:
yep, `handler.close()` use `manager` to close.